### PR TITLE
Create policies for billing API resources

### PR DESCRIPTION
**Pull request by oyving**
More fine-grained access policies for billing API.

### CODE REVIEW

File: controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/Policy.java [hunk 1]
```diff
@@ -142,7 +142,32 @@
...
+
+    /** Ability to update tenant payment instrument */
+    paymentInstrumentUpdate(Privilege.grant(Action.update)
+                                     .on(PathGroup.billingInstrument)
+                                     .in(SystemName.PublicCd)),
+
+    /** Ability to remove your own payment instrument */
+    paymentInstrumentDelete(Privilege.grant(Action.delete)
+                                     .on(PathGroup.billingInstrument)
+                                     .in(SystemName.PublicCd)),
```
Comment (olaaun) [inline controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/Policy.java:160]: What's the benefit of having these as three separate policies?

Comment (oyving) [reply controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/Policy.java:160]: They are assigned differently in roles?

**Comment by olaaun** [comment]
Tests need fixing

File: controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/Policy.java (hunk 1 above)
Comment (olaaun) [reply controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/role/Policy.java:160]: Alright

**Review by olaaun — APPROVED** [review]

**Review by jonmv — COMMENTED** [review]
A bit late to the party here, but looks good to me :)

**Pull request merged**